### ISSUE TITLE
empty report

### ISSUE THREAD

**Issue by ynixon** [kochetkov-ma/allure-server#47]
Using the file from [allure-results.zip](https://github.com/kochetkov-ma/allure-server/files/5868252/allure-results.zip) works well.
Using my own allure report folder works great locally (using allure serve c:\allure )
When uploading my [allure.zip](https://fex.net/s/x4cnfmm) I see no data :(
any idea what is wrong ?

The report was created using https://github.com/ynixon/allure-report-action (version 2.17.2)

**Comment by kochetkov-ma** [comment]
The name of the directory in result archive should start with "allure-" but in [allure.zip](https://fex.net/s/x4cnfmm) it has name "allure"

**Comment by andreyd11** [comment]
@ynixon see the same issue.
Have you sorted what was the cause? Renaming doesn't help in my case.

**Comment by bnagym** [comment]
@ynixon @kochetkov-ma @andreyd11
Hi! I have the same issue. Does anybody find solution for this?
Allure-server seems working fine, however, when I upload my test results (I've checked the name of the .zip archive and folder inside it, both of them starts with "allure-") and try to generate report, it shows nothing in the report url.

Upd: Found solution in https://github.com/kochetkov-ma/allure-server/issues/36. Just rename your folder to "allure-", then make sure that inside .zip archive there is no folders, only .json autotest results.
